sale_commission/models/account_invoice.py

```diff
@@ -70,34 +70,36 @@ def _refund_cleanup_lines(self, lines):
 
 
 class AccountInvoiceLine(models.Model):
+    """Invoice Line inherit to add commissions"""
     _inherit = "account.invoice.line"
 
     @api.model
-    def _default_agents(self):
-        agents = []
-        if self.env.context.get('partner_id'):
-            partner = self.env['res.partner'].browse(
-                self.env.context['partner_id'])
-            for agent in partner.agents:
-                agents.append({'agent': agent.id,
-                               'commission': agent.commission.id})
-        return [(0, 0, x) for x in agents]
-
-    agents = fields.One2many(
-        comodel_name="account.invoice.line.agent",
+    def _default_commissions(self):
+        res = self.env['sale.commission'].get_default_commissions()
+        return [(0, 0, x) for x in res]
+
+    commissions = fields.One2many(
```

Comment:
This field renaming forces to make migration efforts. In this case, it can be only to add `oldname='agents'` attribute.